### PR TITLE
📘  definition / what is CSS

### PR DESCRIPTION
**Pull request by skrzepij**

### CODE REVIEW

**Review by MateuszWasik — APPROVED** [review]

File: content/definitions/czym-jest-html.md [hunk 1]
```diff
@@ -2,5 +2,11 @@
...
 ---
 
-Jest to *język*, rozumiany przez przeglądarki internetowe,
-umożliwiający wyświetlanie treści w odpowiednio zorganizowany sposób.
\ No newline at end of file
+`HyperText Markup Language` - jest to język, rozumiany przez przeglądarki internetowe,
+umożliwiający wyświetlanie treści w odpowiednio zorganizowany sposób.
+
+Tworzymy *szablon* strony z odpowiednich elementów (np `div`, `p`, `header`, `section`) tak, aby całość miała 
+logiczny sens. Możemy to traktować jak strona tytułowa gazecie, pozbawiona jakichkolwiek wizualnych dodatków.
```
Comment (MateuszWasik) [inline content/definitions/czym-jest-html.md]: I think it should be `Mozemy to traktować jak strona tytułowa w gazecie...`

File: content/definitions/czym-jest-html.md [hunk 2]
```diff
@@ -2,5 +2,11 @@
...
+
+Najnowsza oficjalna wersja, to `HTML 5`, która wprowadziła sporo nowości. Jednak minęło już kilka lat od wypuszczenia
```
Comment (MateuszWasik) [inline content/definitions/czym-jest-html.md]: that comma is not needed here I guess `Najnowsza oficjalna wersja to...`

Comment (MateuszWasik) [reply content/definitions/czym-jest-html.md]: I think we can refactor last sentence.

`Najnowsza oficjalna wersja to 'HTML 5', która wprowadziła sporo nowości. Mimo, że od jej premiery minęło wiele lat wciąż jest uważana za standard`


Something like that. What do you think ?

File: content/definitions/czym-jest-css.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+question: Czym jest CSS?
+---
+
+`Cascading Style Sheets` - jest to język, używany w celu określenia części wizualnej strony, 
+wyglądu poszczególnych elementów. Potocznie mówimy, że coś *stylujemy*. 
+
+Na frontendzie, stylowanie najczęściej związane jest z językiem [HTML](/#czym-jest-html) oraz `SVG`.
+
+Najnowszą oficjalną i stale rozwijaną wersją, jest  `CSS3`.
```
Comment (MateuszWasik) [inline content/definitions/czym-jest-css.md:10]: `Najnowszą, oficjalną i stale rozwijaną...`

File: content/definitions/czym-jest-css.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+question: Czym jest CSS?
+---
+
+`Cascading Style Sheets` - jest to język, używany w celu określenia części wizualnej strony, 
+wyglądu poszczególnych elementów. Potocznie mówimy, że coś *stylujemy*. 
+
+Na frontendzie, stylowanie najczęściej związane jest z językiem [HTML](/#czym-jest-html) oraz `SVG`.
```
Comment (MateuszWasik) [inline content/definitions/czym-jest-css.md]: comma is not needed here I think

File: content/definitions/czym-jest-html.md [hunk 5]
```diff
@@ -2,5 +2,11 @@
 question: Czym jest HTML?
 ---
 
-Jest to *język*, rozumiany przez przeglądarki internetowe,
-umożliwiający wyświetlanie treści w odpowiednio zorganizowany sposób.
\ No newline at end of file
+`HyperText Markup Language` - jest to język, rozumiany przez przeglądarki internetowe,
```
Comment (MateuszWasik) [inline content/definitions/czym-jest-html.md]: `jest to język rozumiany przez...` comma is not needed here :) mmmmmmmmm

**Review by MateuszWasik — APPROVED** [review]
Finally you are starting to explain CSS xD we are going to have the best explained CSS in Poland :D

File: content/definitions/czym-jest-html.md (hunk 2 above)
Comment (skrzepij) [reply content/definitions/czym-jest-html.md]: Yeah, I get your point, but what I meant to say is that we don't need anymore to underline that we are using HTML5. Few years before it wasn't so obvious that you are using semantic tags etc, but now it's something natural so saying that you are writing in HTML 5 is something like.. 'I have a flat TV in my house' - ok good for you, but all TVs are now flat xD

Comment (skrzepij) [reply content/definitions/czym-jest-html.md]: will try to refactor it tho

Comment (skrzepij) [reply content/definitions/czym-jest-html.md]: Ok, I shortened it a bit, you can check it out br0000 ✋🏻

File: content/definitions/czym-jest-css.md (hunk 4 above)
Comment (skrzepij) [reply content/definitions/czym-jest-css.md]: indeed 🧐

**Pull request merged**